plugins/node.d/df

```diff
@@ -24,7 +26,7 @@ The following environment variables are used by this plugin:
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
 
-df_entries=$(df -P -l 2>/dev/null | sed -e 1d -e '/\/\//d' -e 's/%//' | sort)
+df_entries=$(df -P -l 2>/dev/null | sed -e '1d' -e '/\/\//d' -e 's/%//' | sort) || true
```

Comment:
Do you have a specific error in mind, that should be tolerated in this line?
I can only imagine a failed `df` invocation. And I guess, we would not want to ignore that?

Reply:
without the quotes it does not work in my sandbox (btw. df_inode has the quotes too)

Reply:
The quotes are fine (but they should not make a difference). I meant `|| true`.